openid-sharedsignals-framework-1_0.md

```diff
@@ -2013,6 +2017,8 @@ The CAEP use cases that set the requirements are described in CAEP Use Cases (TO
 This section provides SSF profiling specifications for the Security Event Token (SET)
 {{RFC8417}} spec.
 
+{{RFC8417}} includes the usage of the `txn` claim in the spec. It is being called out here, to be used in a SET, as both a feedback loop and/or reconciling events between Transmitters and Receivers.
```

Comment:
we need to use more normative language here, I think. E.g., "Transmitters SHOULD set the `txn` value in SETs. If present, this value MUST be unique to the underlying event that has caused the Transmitter to generate the SSF event."

Reply:
Updated verbiage and called out differences from Transmitter underlying txn id's and how those can traverse different SET's